google-cloud-datastore/src/test/java/com/google/cloud/datastore/LatLngTest.java

```diff
@@ -47,35 +43,43 @@ public void testEquals() {
     assertNotEquals(gp1, gp2);
   }
 
-  @Test
+  @Test(expected = IllegalArgumentException.class)
   public void testUpperLatRange() {
-    new LatLng(90, 0);
-    thrown.expect(IllegalArgumentException.class);
-    thrown.expectMessage(INVALID_LAT_MESSAGE);
+    try {
+      new LatLng(90, 0);
+    } catch (IllegalArgumentException e) {
+      assertEquals(INVALID_LAT_MESSAGE, e.getMessage());
+    }
     new LatLng(91, 0);
   }
 
```

Comment:
In LatLngTest for IllegalArgumentException the negative cases is checked twice. That's not needed.
The test will look better if you remove the last constructor call

```
@Test // no expected exception here
public void testUpperLatRange() {
 try {
      new LatLng(90, 0);
    } catch (IllegalArgumentException e) {
      assertEquals(INVALID_LAT_MESSAGE, e.getMessage());
    }
  }
}
```